met-web/src/styles/Theme.ts

```diff
@@ -37,6 +37,11 @@ export const BaseTheme = createTheme({
         action: {
             active: Palette.action.active,
         },
+        info: {
```

Comment:
if all the same why add dark and light

Reply:
Just thought if we wanted to change it later we have access to it

Reply:
Understood. Let's follow the YAGNI clean code principle for now